src/components/Login/ModalOperater/index.tsx

```diff
@@ -4,17 +4,17 @@ import useAuthRequest from "../helpers/authRequest";
 type Props = {
   handleCloseModal: () => void;
   prevStep: () => void;
-  currentStep: number;
+  currentStep: string;
 };
 
 const ModalOperator: React.FC<Props> = ({ handleCloseModal, prevStep, currentStep }) => {
   return (
     <Flex
       width="100%"
       flexDirection="row"
-      justifyContent={currentStep !== 4 ? "space-between" : "end"}
+      justifyContent={currentStep !== "EmailVerification" ? "space-between" : "end"}
```

Comment:
The String "EmailVerification" can be imported from the enum "stepName" you have just created from Login/index file. Normally, we do not use any plain string in any file like this.